sktime/transformations/series/clear_sky.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
```

Comment:
can we rename this file, e.g. to `solar.py` or can we even have a name that is independent on the data domain?

Reply:
Could change it if more solar specific/related transformations come up?

Reply:
we can move this later (after the next release), we need to clean up transformers folder structure anyway...